pkg/scheduler/framework/framework.go

```diff
@@ -39,10 +39,10 @@ const (
 	eventRecorderNameTemplate = "scheduler-framework-%s"
 
 	// The reasons to use for scheduling decisions.
-	pickedByPolicyReason                  = "picked by scheduling policy"
-	pickFixedInvalidClusterReasonTemplate = "cluster \"%s\" is not eligible for resource placement yet: %s"
-	pickFixedNotFoundClusterReason        = "specified cluster \"%s\" is not found"
-	notPickedByScoreReason                = "cluster \"%s\" does not score high enough (affinity score: %d, topology spread score: %d)"
+	pickedByPolicyReason                   = "picked by scheduling policy"
```

Comment:
why is this message not start with capital letter?

Reply:
It is used at the of [resourceScheduleSucceededMessageFormat ](https://github.com/Azure/fleet/blob/754df442ec85dbc599c22c62a1201960e02836f7/pkg/scheduler/framework/frameworkutils.go#L26C2-L26C114) and [resourceScheduleSucceededWithScoreMessageFormat](https://github.com/Azure/fleet/blob/754df442ec85dbc599c22c62a1201960e02836f7/pkg/scheduler/framework/frameworkutils.go#L27). I can delete and just add the statement to the end of these.

Reply:
next PR then. We should group all the reason in one file, it's scattered now.

Reply:
Addressed on #845